bot/cogs/verification.py

```diff
@@ -32,26 +59,408 @@
 <#{constants.Channels.bot_commands}>.
 """
 
-BOT_MESSAGE_DELETE_DELAY = 10
+# Sent via DMs to users kicked for failing to verify
+KICKED_MESSAGE = f"""
+Hi! You have been automatically kicked from Python Discord as you have failed to accept our rules \
+within `{KICKED_AFTER}` days. If this was an accident, please feel free to join again.
+"""
```

Comment:
I think we should try make it as easy as possible for users to re-join in this message.

```suggestion
KICKED_MESSAGE = f"""
Hi! You have been automatically kicked from Python Discord as you have failed to accept our rules \
within `{KICKED_AFTER}` days. If this was an accident, please feel free to join us again at https://discord.gg/python.
"""
```

Reply:
Yea, good idea. I'm tempted to define it in the config, probably under `Guild.invite`.

Reply:
Added invite in 53189e8, but I had to put it on the next line. The invite does not embed properly if followed by a dot.
![image](https://user-images.githubusercontent.com/44734341/91332427-745fc700-e7cc-11ea-9612-4e4c951d9b6d.png)